docs/1.2. Behaviour.md

```diff
@@ -4,7 +4,7 @@ The Sink Metadata Processing API provides a mechanism to change the settings ass
 
 ## Senders
 
-`/media-profiles` allows the client to pass an array of Media Profiles to a Sender and let a third-party which controls it choose what exact format to use. A client can also pass a single Media Profile to specify the exact format.
+`/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the device which controls it choose what exact format to use. A client can also pass a single Media Profile to specify the exact format.
```

Comment:
`Device` is quite physical. I'm not sure if we should avoid terms which are 100 % hardware or software related. If we should, then I'd propose `underlying component`.